.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -2,53 +2,53 @@ name: Bug report
 description: Report something that is broken or incorrect
 labels: bug
 body:
-- type: textarea
-  id: description
-  attributes:
-    label: Description of the bug
-    description: A clear and concise description of what the bug is.
-  validations:
-    required: true
-- type: textarea
-  id: command_used
-  attributes:
-    label: Command used and terminal output
-    description: Steps to reproduce the behaviour. Please paste the command you used
-      to launch the pipeline and the output from your terminal.
-    render: console
-    placeholder: '$ nextflow run ...
-
-
-      Some output where something broke
-
-      '
-- type: textarea
-  id: files
-  attributes:
-    label: Relevant files
-    description: 'Please drag and drop the relevant files here. Create a `.zip` archive
-      if the extension is not allowed.
-
-      Your verbose log file `.nextflow.log` is often useful _(this is a hidden file
-      in the directory where you launched the pipeline)_ as well as custom Nextflow
-      configuration files.
-
-      '
-- type: textarea
-  id: system
-  attributes:
-    label: System information
-    description: '* Nextflow version _(eg. 23.04.0)_
-
-      * Hardware _(eg. HPC, Desktop, Cloud)_
-
-      * Executor _(eg. slurm, local, awsbatch)_
-
-      * Container engine: _(e.g. Docker, Singularity, Conda, Podman, Shifter, Charliecloud,
-      or Apptainer)_
-
-      * OS _(eg. CentOS Linux, macOS, Linux Mint)_
-
-      * Version of HoffLab/caganalyzer _(eg. 1.1, 1.5, 1.8.2)_
-
-      '
+  - type: textarea
+    id: description
+    attributes:
+      label: Description of the bug
+      description: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: command_used
+    attributes:
+      label: Command used and terminal output
+      description: Steps to reproduce the behaviour. Please paste the command you used
+        to launch the pipeline and the output from your terminal.
```

Comment:
Supposed to be on a separate line?

Reply:
Sometimes if a line is too long (I think 100 characters is the limit) they "wrap" it to the next line so that it is more readable. If this is not done manually, I think it will actually be done automatically when committing as well!